@ecomplus/storefront-app/src/lib/sync-cart-to-api.js

```diff
@@ -82,6 +83,9 @@ const prepareCart = cart => {
   } else {
     url = '/carts.json'
     method = 'POST'
+    if (router && router.currentRoute && router.currentRoute.name === 'confirmation' && completed === undefined) {
```

Comment:
```suggestion
    if (router.currentRoute.name === 'confirmation' && !completed) {
```

1. Não tem como `router` ser indefinido aqui;
2. `completed = false` não deve passar também;